src/libcrun/status.c

```diff
@@ -71,22 +96,62 @@ get_state_directory_status_file (const char *state_root, const char *id)
   return ret;
 }
 
+static int
+read_pid_stat(pid_t pid, struct pid_stat *st, libcrun_error_t *err)
+{
+  int ret;
+  FILE *f;
+  cleanup_free char *pid_stat_file = NULL;
+
+  ret = xasprintf ((char **)&pid_stat_file, "/proc/%d/stat", pid);
+  if (UNLIKELY (ret == -1))
+    return crun_make_error (err, errno, "xasprintf failed");
+
+  f = fopen (pid_stat_file, "r");
+  if (f == NULL)
+    return crun_make_error (err, errno, "open state file %s", pid_stat_file);
+
+  ret = fscanf(f,"%d %255s %c %d %d %d %d %d %u %lu %lu %lu %lu %lu %lu %ld %ld %ld %ld %ld %ld %llu",
```

Comment:
same nit here: an empty space after function name.

Reply:
thank you for the feedback! `make syntax-check` fails here with:
```
do not use *scanf, atof, atoi, atol, atoll or atoq
```
@giuseppe what would be a good alternative of `fscanf`?